### PR TITLE
bugfix/accessibility: img tabindex control

### PR DESCRIPTION
**Pull request by oliverfoster**
removed image tabindexes and injected the image alt tag as a tabbable area before the title.

### CODE REVIEW

**Comment by moloko** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

File: templates/narrative.hbs [hunk 1]
```diff
@@ -64,7 +64,7 @@
             <div class="narrative-slider clearfix">
                 {{#each _items}}
                 <div class="narrative-slider-graphic {{#if visited}}visited{{/if}}">
-                    <img src="{{_graphic.src}}" aria-label="{{_graphic.alt}}" tabindex="0"/>
+                    <img src="{{_graphic.src}}" alt="{{_graphic.alt}}""/>
```
Comment (hrajotia) [inline templates/narrative.hbs]: @oliverfoster need to remove the extra double-quote at the end.

**Comment by hrajotia** [comment]
:+1:

**Comment by moloko** [comment]
+1 good spot @hrajotia!

**Comment by oliverfoster** [comment]
https://adaptlearning.atlassian.net/browse/ABU-1140

**Pull request merged**